libs/orchestrator-ui-components/src/lib/components/Table/columns.ts

```diff
@@ -15,28 +15,23 @@ export type TableColumnsWithExtraNonDataFields<T> = TableColumns<T> & {
     };
 };
 
-export enum SortDirection {
-    Asc = 'ASC',
-    Desc = 'DESC',
-}
-
 export type DataSorting<T> = {
-    columnId: keyof T;
-    sortDirection: SortDirection;
+    columnId?: keyof T;
```

Comment:
For dataSorting I would say that both columnId and sortDirection should be always defined. In case of not sorting, make the whole prop DataSorting optional.

Reply:
Agreed. Is there a case when not sorting though? I have a default sorting on products of name/ASC now